### PR TITLE
Added toggle include crew keybind

### PR DESCRIPTION
**Pull request by CreepPork**
**When merged this pull request will:**
- Add a key bind to toggle include crew

### CODE REVIEW

File: @AresModAchillesExpansion/addons/settings_f/scripts/keybindings.sqf [hunk 1]
```diff
@@ -142,3 +142,23 @@
...
+			((findDisplay 312) displayCtrl 2801) cbSetChecked false;
+		}
+		else
+		{
+			Achilles_var_deleteCrewOnSpawn = nil;
+			((findDisplay 312) displayCtrl 2801) cbSetChecked true;
+		};
+	},
+	"",
+	[DIK_LMENU, [false, false, false]]
```
Comment (oOKexOo) [inline @AresModAchillesExpansion/addons/settings_f/scripts/keybindings.sqf]: ALT as default will inevitably conflict with combinations such as ALT+2xLMB. Players should define their own binding if they gonna use that feature.

Comment (oOKexOo) [inline @AresModAchillesExpansion/addons/settings_f/scripts/keybindings.sqf]: ALT as a default creates conflicts with existing keybindings that use ALT. I would set no keybinding, such that the players has to specify what suits for him.

**Comment by CreepPork** [comment]
I would prefer it to work when the set key is released then it should go back to it's previous position just like in Eden.

**Comment by CreepPork** [comment]
As I can’t review my own PR, it’s fine.

**Pull request merged**